data/remnant/remnant jobs.txt

```diff
@@ -223,6 +224,115 @@ mission "Remnant: Bounty 5"
 
 
 
+mission "Remnant: Logistics Civil Engineering"
+	name "Civil delivery to <planet>"
+	description `Engineers on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant civil" 30 80
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
+		not "flagship planet: Ssil Vida"
```

Comment:
```suggestion
```

Reply:
I'm guessing the point is that this redundant because the Ssil Vida is excluded in the source statement. Removed from all jobs.